distribution/lib/Standard/Base/0.0.0-dev/src/Errors/Common.enso

```diff
@@ -576,3 +577,19 @@ type Response_Too_Large
        Convert the Java exception to an Enso dataflow error.
     handle_java_exception ~action =
         Panic.catch ResponseTooLargeException action (cause-> Error.throw (Response_Too_Large.Error cause.payload.getLimit))
+
+## Indicates that a Float was used as a Dicionary key.
+type Float_Used_As_Dictionary_Key
```

Comment:
This type feels slightly redundant with already existing `Floating_Point_Equality`.

However, I do like that we get a more precise message ('Float used as dictionary key'). What about merging this into the `Floating_Point_Equality` error, perhaps by distinguishing different constructors (we already have multi-constructor errors, e.g. `Invalid_Value_Type`).

Reply:
Done -- moved `Floating_Point_Equality` into `Base`.